Modules/Core/Common/src/itkMersenneTwisterRandomVariateGenerator.cxx

```diff
@@ -96,9 +96,9 @@ MersenneTwisterRandomVariateGenerator::CreateInstance()
 MersenneTwisterRandomVariateGenerator::Pointer
 MersenneTwisterRandomVariateGenerator::New()
 {
-  MersenneTwisterRandomVariateGenerator::Pointer obj = MersenneTwisterRandomVariateGenerator::CreateInstance();
+  MersenneTwisterRandomVariateGenerator::Pointer obj = Self::CreateInstance();
 
-  obj->InitializeWithoutMutexLocking(MersenneTwisterRandomVariateGenerator::GetNextSeed());
+  obj->InitializeWithoutMutexLocking(Self::GetNextSeed());
```

Comment:
Technically, it isn't even necessary to add `Self::` to internal calls to its own static member functions. But this is the way we like it, right?  😇